config/upcloud/group_vars/k8s_cluster/ck8s-k8s-cluster-upcloud.yaml

```diff
@@ -0,0 +1,30 @@
+etcd_deployment_type: kubeadm
+
+## Repo for UpClouds csi-driver: https://github.com/UpCloudLtd/upcloud-csi
+## To use UpClouds CSI plugin to provision volumes set this value to true
+## Remember to set UPCLOUD_USERNAME and UPCLOUD_PASSWORD
+upcloud_csi_enabled: true
+upcloud_csi_enable_volume_snapshot: true
+
+#upcloud_csi_controller_replicas: 1
+## Override used image tags
+# upcloud_csi_provisioner_image_tag: "v3.1.0"
+# upcloud_csi_attacher_image_tag: "v3.4.0"
+# upcloud_csi_resizer_image_tag: "v1.4.0"
+# upcloud_csi_plugin_image_tag: "alpha"
+# upcloud_csi_node_image_tag: "v2.5.0"
+upcloud_tolerations:
+  - key: elastisys.io/node-type
+    operator: Exists
+    effect: NoSchedule
```

Comment:
We don't want Elastisys specifics in this repository.

Reply:
I'll remove them from here and add it in our templates.